web/posts/2017-07-25-making-time-for-side-projects.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: Making Time for Side Projects
+tags: development, life
+---
+
+I have about a million things I'd like to do at any given moment. If I sit
+down and try to choose something to spend my time on, it's fairly
+overwhelming. There are blog posts I could write, books I could read, movies
+and TV shows I could watch, or any number of ways I could continue my own
+education via videos or articles. And then there are the side projects. So so
+many side projects.
+
+The problem with side projects is that you can't complete them. No matter what
+you think, side projects aren't done. You might be able to finish reading a
+book or watching a run of a TV show, but you'll never finish that "small" tool
+you've been meaning to write. There will _always_ be something to improve,
+something to tweak, something to fix.
+
+I decided a while ago to write my own microblog. This "small" app was supposed
+to serve two different purposes:
+
+1. To take back ownership of my writing and host it under my own domain and
+   in my own database.
+2. To act as an excuse to learn [Yesod] so that I could start on my _real_
+   side project.
+
+[Yesod]: https://www.yesodweb.com/
+
+[My microblog][featureless-void] is now 7 months old and has seen [133
+commits][featureless-void-commits] to master (at the time of this writing) and
+isn't _close_ to being "finished". And this isn't even the thing I _want_ to
+work on as my side project. I've spent so much time working on The Small Thing
+that I've barely even _started_ on The Real Thing.
+
+[featureless-void]: https://github.com/gfontenot/featureless-void
+[featureless-void-commits]: https://github.com/gfontenot/featureless-void/commits/master
+
+So, presented with infinite things to do, an infinite amount of time that
+those things will take up, and a spectacularly finite amount of time to do
+them: how the shit do I actually get things done?
+
+The solution I'm going to experiment with is to explicitly set aside a part of
+my week that I tend to otherwise "waste"[^1], and dedicate it to working on a
+side project. For me, this means Sunday mornings (before noon). I expect
+_most_ of that time to be spent working on my main side project (a workout
+tracking app I'm currently calling Swole), but I'm also not going to stop
+myself from working on projects like [my microblog][featureless-void], [my
+dotfiles][dotfiles], or [any][reflow] of [my other][argo] [open source][runes]
```

Comment:
Can you have at least one non-link word between each linked word? Otherwise it's hard to tell that there's >1 link here. Maybe make it look like this:

[any]() of [my]() other [open]() source [projects]()